src/services/target/target.js

```diff
@@ -0,0 +1,25 @@
+import endpoints from 'constants/endpoints';
+import { api } from 'services/api';
+
+const targetApi = api.injectEndpoints({
+  endpoints: builder => ({
+    createTarget: builder.mutation({
+      query: target => ({
+        url: endpoints.TARGET,
+        method: 'POST',
+        body: { target },
+      }),
+    }),
+    getTarget: builder.mutation({
```

Comment:
`builder.mutation` is used to send data to the server, to get data you should use `builder.query`:

https://redux-toolkit.js.org/rtk-query/usage/queries
```suggestion
    getTarget: builder.query({
      query: () => ({
        url: endpoints.TARGET
      })
    })
```

Reply:
I will make this change.